spec/view/management_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe "Management", type: :feature do
+  describe 'index page' do
+    before do
+      ENV['REGISTRY_HOST'] = ''
+      ENV['REGISTRY_URI'] = 'yalelibraryit/dc-management'
+      ENV['MANAGEMENT_TAG'] = 'v1.2.3'
```

Comment:
Since we're in the v1.x.x series for both Management and Solr right now, could we have these tags be something more obviously fake, so that we don't get confused? Like `v1Beta.Gamma` or something?

Reply:
Addressed in latest commit